book/src/users/monitoring.md

```diff
@@ -39,9 +39,29 @@ htop
 ```
 
 ## Metrics
+[Metrics setup with docker](#metrics-setup-with-docker) (Recomended)
+
 [Metrics setup no docker](#metrics-setup-no-docker)
 
-[Metrics setup with docker](#metrics-setup-with-docker)
+## Metrics setup with docker
+1. [Install Docker](https://docs.docker.com/engine/install/).
+
+2. From the `metrics` directory run:
+```sh
+docker compose up
+```
+
+3. Start your Trin process exposing metrics:
+```sh
+cargo run -p trin -- --enable-metrics-with-url 0.0.0.0:9100 --web3-http-address http://0.0.0.0:8545 --web3-transport http
+```
+- The addresses must be bound to 0.0.0.0, because 127.0.0.1 only allows internal requests to
+  complete, and requests from docker instances are considered external.
+- The `--enable-metrics-with-url` parameter is the address that Trin exports metrics to, and should be equal to the port to which your Prometheus server is targeting at the bottom of `prometheus/prometheus.yml`
+- The `--web-transport http` will allow Grafana to request routing table information from Trin via JSON-RPC over HTTP
+
+4. Navigate to http://localhost:3000/d/trin-metrics/trin-metrics. Use `admin`/`admin` to login.
+
 
 ## Metrics setup no docker
```

Comment:
If you want you can also delete the `## Metrics setup no docker`, I don't think we should keep it as
- it is more complex to maintain especially as people are unlikely to use it so nobody will know how it works so they can fix it
- the docker solution is just better

No worries if you don't want to include that in this PR, we can always remove it on a different date

Reply:
I'll go ahead and remove it, that way the structure of the new `metrics` dir can be the same as the one in `trin-execution`